src/tracker-component.js

```diff
@@ -0,0 +1,40 @@
+/* @flow */
+
+import { getClientID } from '@paypal/sdk-client/src';
+
+const user = { id: undefined, name: undefined };
+
+const track = (trackingType, trackingData) => {
+    const encodeData = data => encodeURIComponent(btoa(JSON.stringify(data)));
+
+    return new Promise((resolve, reject) => {
+        const img = document.createElement('img');
+        img.style.display = 'none';
+        img.src = `https://api.keen.io/3.0/projects/5b903f6ec9e77c00013bc6a7/events/${ trackingType }?api_key=700B56FBE7A2A6BD845B82F9014ED6628943091AD5B0A5751C3027CFE8C5555448C6E11302BD769FCC5BDD003C3DE8282C4FC8FE279A0AAC866F2C97010468197B98779B872EFD7EE980D2986503B843DA3797C750DAA00017DC8186078EADA6&data=${ encodeData(
+            {
+                ...trackingData,
+                userId:   user.id,
+                userName: user.name,
+                clientId: getClientID()
+            }
+        ) }`;
+        img.addEventListener('load', resolve);
+        img.addEventListener('error', reject);
+        document.body.appendChild(img);
+    });
+};
+
+const trackCartEvent = (cartEventType, trackingData) =>
+    track('cartEvent', { ...trackingData, cartEventType });
+
+export const Tracker = {
```

Comment:
Can we call this `CartTracker` maybe so it's obvious what the namespace represents? Given the number of other trackers we're surfacing in this sdk.

Reply:
which are the other trackers on the sdk? Any easy way to know that?

Reply:
Well muse already does tracking, we have fpti tracking, cal tracking, etc. Just want to make sure any public `paypal.*` names aren't too ambiguous before we set them in stone.